Setup/Product.wxs

```diff
@@ -895,13 +886,13 @@
         <RegistryValue Name="gitssh" Value="[CURRENT_SSHCLIENT]" Root="HKCU" Key="$(var.AppRegKey)" Type="string" />
       </Component>
       <Component Id="plink.reg" Guid="*">
-        <RegistryValue Name="plink" Value="[INSTALLDIR]PuTTY\plink.exe" Root="HKCU" Key="$(var.AppRegKey)" Type="string" />
+        <RegistryValue Name="plink" Value="[ProgramFilesFolder]plink.exe" Root="HKCU" Key="$(var.AppRegKey)" Type="string" />
       </Component>
       <Component Id="pageant.reg" Guid="*">
-        <RegistryValue Name="pageant" Value="[INSTALLDIR]PuTTY\pageant.exe" Root="HKCU" Key="$(var.AppRegKey)" Type="string" />
+        <RegistryValue Name="pageant" Value="[ProgramFilesFolder]pageant.exe" Root="HKCU" Key="$(var.AppRegKey)" Type="string" />
       </Component>
       <Component Id="puttygen.reg" Guid="*">
-        <RegistryValue Name="puttygen" Value="[INSTALLDIR]PuTTY\puttygen.exe" Root="HKCU" Key="$(var.AppRegKey)" Type="string" />
+        <RegistryValue Name="puttygen" Value="[ProgramFilesFolder]puttygen.exe" Root="HKCU" Key="$(var.AppRegKey)" Type="string" />
       </Component>
```

Comment:
We don't need this either, since we're no longer ship or install it, we mustn't register those either.

Reply:
I thought these set the path to the PuTTY binaries in the GE registry, which still is needed.
There is detection of PuTTY path in the GE settings when setting PuTTY. If PuTTY is removed completely from the installer and the user configures this in the setting, it is easier to handle. But I have been flamed when suggesting that previously. (I still prefer that myself.)

For 3.5 I would like to update the binaries, there is a 0.76 version coming soon that I hope will be in time for a 3.5.2 release.
The tuning of the path should probably be in 3.5.2 too (to select ProgramFiles/PuTTY of the distributed version).

> On the settings page we can show the actual settings, if we can find those,

When selecting PuTTY the hardcoded directories (PUTTY, TortoiseGit, TortoiseSVN) are searched, you can set manually after that.

> or a link to the wiki page with instructions.

A link could be put there regardless
But I want to keep the complexity for the change to a minimum.

Reply:
> For 3.5 I would like to update the binaries, there is a 0.76 version coming soon that I hope will be in time for a 3.5.2 release.
> The tuning of the path should probably be in 3.5.2 too (to select ProgramFiles/PuTTY of the distributed version).

👍 
Yes, we're few things for 3.5.2. With 27K downloads I think we can assume the main lot of bugs has been reported.



> If PuTTY is removed completely from the installer and the user configures this in the setting, it is easier to handle.

That'd be my preference.

Reply:
Removed PuTTY from the installer (it will be much easier to test this now...).
Documentation will have to be changed too.

I will consider a migration, may open the settings if pointing to the removed binaries, but want to have it really simple.